### PR TITLE
Document schema policy (#393).

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

File: ansible/roles/kraken.config/files/schemas/README.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+additional properties it finds in a document (i.e. these documents are still considered valid).
+
+k2 configuration files should be backwards compatible but may not be forward compatible. That is a configuration file generated with k2 version x.y should be compatible with version k2 version x+1. On the other hand, a configuration file generated with k2 version x.y may not be compatible with k2 version x-1.
+
+Changes may be breaking or non-breaking. Changes which extend the schema, by for example, adding an __optional__ property, are non-breaking. Changes which make a __required__ property __optional__ are also non-breaking. All other syntactic, _and_ semantic, changes are generally breaking and should be avoided.
+
+If it becomes necessary to make a breaking change, a new schema with a new version must be created. Previous schema versions should be supported for at least two k2 releases or 3 months (whichever is longer). During this time, attempts to use configuration files generated by older k2 releases will fail and a depreciation notice will be printed. In order to continue using these configuration files the --force flag must be passed to k2cli.
```
Comment (joejulian) [inline ansible/roles/kraken.config/files/schemas/README.txt]: Just one spelling error, it's "deprecation" not "depreciation".

**Comment by coffeepac** [comment]
@NathanZook separate issue.  we do want to provide that though.

**Comment by davidewatson** [comment]
Please retest this

**Comment by davidewatson** [comment]
retest this please

**Pull request merged**